automationTest/src/test/java/com/owncloud/android/test/ui/models/LoginForm.java

```diff
@@ -0,0 +1,91 @@
+package com.owncloud.android.test.ui.models;
+
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.PageFactory;
+
+import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.android.AndroidElement;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+import io.appium.java_client.pagefactory.AppiumFieldDecorator;
+
+public class LoginForm {
+	final AndroidDriver driver;
+	
+	@CacheLookup
+	@AndroidFindBy(uiAutomator = "new UiSelector().description(\"Server address\")")
+	private AndroidElement hostUrlInput;
+	
+	@CacheLookup
+	@AndroidFindBy(uiAutomator = "new UiSelector().description(\"Username\")")
```

Comment:
Please (if it is possible) use a string reference instead of a literal.

getResources().getString(R.string.auth_username)